vulcanai2/tests/models/test_layers.py

```diff
@@ -3,25 +3,17 @@
 import torch
 from vulcanai2.models.layers import *
 
-class TestBaseLayer:
+class TestBaseUnit:
     @pytest.fixture
-    def baselayer(self): 
-        return BaseLayer()
-
-    # TODO: Needs fixing assigning values to NoneType
```

Comment:
Why was this deleted?

Reply:
Rename to `BaseUnit`

Reply:
sorry, I was referring to the #TODO comment below

Reply:
the test function of modifying variables in baseunit

Reply:
those variables are dummy and are not being used. Although I would like to take a look at them once while working on the other layer classes.

Reply:
ok let's make a not of that through docs in the BaseUnit class